packages/pulpcore/python-dynaconf/python-dynaconf.spec

```diff
@@ -49,6 +52,9 @@ rm -rf %{pypi_name}.egg-info
 %{python3_sitelib}/%{pypi_name}-%{version}-py%{python3_version}.egg-info
```

Comment:
```suggestion
%{python3_sitelib}/%{pypi_name}-%{version}rc1-py%{python3_version}.egg-info
```

Reply:
the %autosetup line above also needs to become `%{version}rc1` above, but I can't comment on it as it's outside the current diff